src/platform/backends/shared/base_virtual_machine.cpp

```diff
@@ -59,7 +59,7 @@ constexpr auto yes_overwrite = true;
 
 void assert_vm_stopped(St state)
 {
-    assert(state == St::off || state == St::stopped);
+    assert(state == St::off || state == St::stopped || state == St::unavailable);
```

Comment:
Why add this here? I don't think we should allow taking or restoring snapshots on unavailable instances. Those are the only two operations using this.